Ports/.port_include.sh

```diff
@@ -889,7 +901,7 @@ parse_arguments() {
         return
     fi
     case "$1" in
-        fetch|patch|shell|configure|build|install|installdepends|clean|clean_dist|clean_all|uninstall|showproperty|generate_patch_readme)
+        fetch|patch|shell|configure|build|install|installdepends|clean|clean_dist|clean_all|uninstall|showproperty|generate_patch_readme|do_generate_available_ports)
```

Comment:
`generate_available_ports` to be consistent with `generate_patch_readme`

Reply:
that was event not an inconsistency but a mistake! 😝